### PR TITLE
Improve visibility of captions

### PR DESCRIPTION
**Pull request by WatanabeToshimitsu**
## What?
Improve visibility of captions

## Why?
Improve UX

## Screenshot or video
- Before
image

- After
image

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-scene-viewer/pull/89#" title="Author self-approved">WatanabeToshimitsu</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-scene-viewer).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-scene-viewer/blob/main/OWNERS)~~ [WatanabeToshimitsu]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by d-hayashi** [comment]
現在時刻のキャプションのパネルの文字ってもう少し大きくできます？  
あとこのパネルではタイムスタンプの表示は不要な気がしてきました

**Comment by WatanabeToshimitsu** [comment]
@d-hayashi 
レビューを反映しました　いかがでしょうか

**Review by d-hayashi — APPROVED** [review]
見やすくなっていい感じだと思います！ 
LGTM

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-scene-viewer/pull/89#pullrequestreview-945204360" title="Approved">d-hayashi</a>*, *<a href="https://github.com/dataware-tools/app-scene-viewer/pull/89#" title="Author self-approved">WatanabeToshimitsu</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-scene-viewer).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-scene-viewer/blob/main/OWNERS)~~ [WatanabeToshimitsu,d-hayashi]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
